CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to Econ-ARK
+
+### Welcome!
+
+Thank you for considering contributing to Econ-ARK!  We're a young project with a small but committed community that's hoping to grow while maintaining our friendly and responsive culture.  Whether you're an economist or a technologist, a writer or a coder, an undergrad or a full professor, a professional or a hobbyist, there's a place for you in the Econ-ARK community. 
+
+We're still creating our contribution infrastructure, so this document is a work in progress.  If you have any questions please feel free to @ or otherwise reach out project manager [Shauna](https://github.com/shaunagm), or lead developers [Chris](https://github.com/llorracc) and [Matt](https://github.com/mnwhite).  If you prefer to connect through email, you can send it to __econ-ark at jhuecon dot org__.
```

Comment:
We should choose a more specific address than econ-ark@jhuecon.org.  Probably we should think about deliberately choosing a few email addresses that will have well-defined routings.  I've just emailed Andrij to ask him to set up the infrastructure for receiving mail at @econ-ark.org.  Maybe we need "contactus@econ-ark.org", "help@econ-ark.org", "issues@econ-ark.org" -- I don't know, we'll have to think about it.

Reply:
Yeah, we ought to think through the classes of things people might want to contact us for, and see if it makes sense to route them to different email addresses.  And I like the idea of having @econ-ark.org addresses, feels fancy.  :)  

Do you want to wait to merge this PR, or come back and change the contact in this doc once it's ready?